.github/workflows/sql-test-and-build-workflow.yml

```diff
@@ -20,25 +20,32 @@ on:
 jobs:
   build:
     strategy:
+      # Run all jobs
+      fail-fast: false
```

Comment:
Why was fail fast was disabled here?

Reply:
This was mainly for my local dev to run all CI even if one fails. I think it would be useful for future devs to figure out if something is breaking just because of OS, java-version, etc. by running all combinations by default.